src/utils/components/Consoles/components/vnc-console/VncConsole.tsx

```diff
@@ -97,17 +97,24 @@ export const VncConsole: FC<VncConsoleProps> = ({
         }
         const clipboardText = await navigator?.clipboard?.readText?.();
         const text = clipboardText || pasteText.current;
-        const lastItem = text.length - 1;
-        for (let i = 0; i < text.length; i++) {
-          const char = text[i];
-          const shiftRequired = isShiftKeyRequired(char);
-          await sleep(50);
-          shiftRequired && this.sendKey(KeyTable.XK_Shift_L, 'ShiftLeft', true);
-          this.sendKey(char.charCodeAt(0));
-          shiftRequired && this.sendKey(KeyTable.XK_Shift_L, 'ShiftLeft', false);
-          i === lastItem &&
-            clipboardText?.charCodeAt(lastItem) === 13 &&
-            this.sendKey(KeyTable.XK_KP_Enter);
+        for (const codePoint of text) {
+          const codePointIndex = codePoint.codePointAt(0);
+          if (codePointIndex === 10) {
+            this.sendKey(KeyTable.XK_Return);
+          } else if (codePointIndex === 9) {
+            this.sendKey(KeyTable.XK_Tab);
+          } else if (codePointIndex >= 0x20 && codePointIndex <= 0xff) {
```

Comment:
Please give all numbers a constant with a meaningful name to improve code readability